includes/class-sensei-autoloader.php

```diff
@@ -82,6 +82,7 @@ public function __construct() {
 			new Sensei_Autoloader_Bundle( 'reports/overview/services' ),
 			new Sensei_Autoloader_Bundle( 'admin/home/quick-links' ),
 			new Sensei_Autoloader_Bundle( 'admin/home/help' ),
+			new Sensei_Autoloader_Bundle( 'admin/home/promo-banner' ),
```

Comment:
I'm using this for now in my PR #5820 as well, but we might want to move away from these bundlers. I think it slows down the autoloader because every time the autoloader is used, it has to go through each of these and do 2 `file_exists` checks.

Reply:
Yeah.. At first I started to manually add the entries one by one but later used this for simplicity. You are right about performance... Food for thought: It would be nice if we could autogenerate the list of files instead of manually adding them.

Just to confirm I understand, you are suggesting removing this bundle and adding all the files one by one below in the list of files. Right?